docs/candidate-app/candidates.md

```diff
@@ -0,0 +1,47 @@
+# Candidates
+
+## Mock Data
+By default, mock data includes the user `first.last@example.com` with the password `password` that can be used to test the candidate app.
+
+## Creating a New Candidate
+A candidate can log in to the candidate app either by having the admin give them a registration key or registering an account for them.
+
+### Using Registration Key
+To use the registration code, you need to navigate to the Strapi's admin area (http://localhost:1337), select the "Content Manager" tab, and choose the "Candidates" tab from the dropdown. If a candidate you want to log in as doesn't exist yet, you can use the "Create new entry" button to create one.
+
+Afterward, edit the desired candidate and choose a secure and random value for the registrationKey field. Then, save the change by clicking the "Save" button. The chosen registrationKey should be shared, and then the candidate can navigate to http://localhost:5173/candidate/register to create their account.
+
+### Manually
+To create the user manually, you need to navigate to the Strapi's admin area (http://localhost:1337), select the "Content Manager" tab, and choose the "User" tab from the dropdown. 
+
+You can create a new user using the "Create new entry" button. The following fields should be set:
+- username (any desired username)
+- email (any desired email, it'll be used for logging in)
+- password (any strong password)
+- confirmed => True
+- blocked => False
+- role => Authenticated
+- candidate => the desired candidate the user is linked to
+
+Then, use the "Save" button and you should be able to log in as the candidate at http://localhost:5173/candidate.
+
+## Resetting Password
```

Comment:
We will look into it. I made a task for it [here](https://github.com/orgs/OpenVAA/projects/2/views/1?pane=issue&itemId=49184691)

Reply:
Tested this, creating a new user like the following:
<img width="895" alt="image" src="https://github.com/OpenVAA/voting-advice-application/assets/10120342/c9b7cf47-b2af-4b96-b1dc-db52011ab97e">
and I receive the password reset email. Are you sure you're not accidentally entering the email into the username field? Would you be able to screenshot (similarly to the picture above) both the created candidate and the user you created so I could try to reproduce this further?

Reply:
Hi! I tested it again but couldn't reproduce the issue even though I tried all permutations of the field values I could think of. 😄 

It remains a mystery what caused the issue earlier. I don't think was a mixup between username and email because the email field is both required and validated, though.